app/views/podcast_episodes/show.html.erb

```diff
@@ -28,7 +28,7 @@
 
 <% end %>
 
-<div class="podcast-episode-container" data-meta="<%= @episode.decorate.mobile_player_metadata.to_json %>">
+<div id="main-content" class="podcast-episode-container" data-meta="<%= @episode.decorate.mobile_player_metadata.to_json %>">
```

Comment:
Just a small change to request - can we also change this to a `<main>` element (and the closing tag)?

This will help any users of assistive technology who navigate by landmarks on the page 🙂